packages/widgets/src/Pagination/PageSizeSelect.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+import i18n from '@dhis2/d2-i18n'
+import { SingleSelect, SingleSelectOption } from '@dhis2/ui-core'
+import { spacers } from '@dhis2/ui-constants'
+import { sectionStyles, selectStyles } from './Pagination.styles'
+
+const PageSizeSelect = ({ dataTest, pageSize, pageSizes, onChange }) => (
+    <div className="section" data-test={`${dataTest}-pagesize`}>
+        {i18n.t('Items per page')}
+        <SingleSelect
+            dense
+            selected={pageSize.toString()}
+            onChange={({ selected }) => onChange(parseInt(selected))}
```

Comment:
The `parseInt` call lacks the radix (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt)

Reply:
Good catch, I always forget them. Didn't we used to have a linter rule that would pick this up? 2286e50

Reply:
There are linter rules for this (airbnb enables it for instance), so I assume we've just not enabled that rule.